lib/browser.js

```diff
@@ -200,32 +147,16 @@ class Browser {
   onResult (result) {
     if (result.length) {
       return result.forEach(this.onResult, this)
```

Comment:
This `return` is confusing and unneeded.

Reply:
True, fixed